ci-operator/jobs/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master-presubmits.yaml

```diff
@@ -121,3 +121,76 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-gcp,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-gcp-ovn
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-kni-cnf-features-deploy-master-e2e-gcp-ovn
```

Comment:
needs ``` optional: true``` right? we don't want this to necessarily block PRs

Reply:
Right. Sorry, did not notice your reviews before.